app/src/main/res/values/strings.xml

```diff
@@ -29,12 +29,12 @@
          For example, in Russian, Android's plural quantity "one" actually refers to "any number ending on 1 but not ending in 11".
          So while in English the extra non-plural form seems unnecessary duplication, it is necessary to give translators enough flexibility.
          In Catima, we use the plain string when meaning exactly 1, and otherwise use the plural forms -->
-    <string name="deleteTitle">Delete card</string>
+    <string name="deleteTitle">Deletion</string>
```

Comment:
Please read the i18n oddness note as to why it makes no sense to make deleteTitle so different to deleteCardsTitle.